CONTRIBUTING.md

```diff
@@ -1,65 +1,99 @@
-- One person only is allowed to edit a given PR (unless given permission to other committers).
-- A PR should only create/modify content within the scope of a single workstream.
-- A PR should only add/change content related to the issue associated with the PR
-
+  - using GitHub predefined keyword "**Fixes Issue#**" will automatically close the issue which the PR is resolving when it is merged.
+- One person only is allowed to edit a given PR unless given permission to other contributor to contribute directly into the PR.
+- A PR is recommended to only create/modify content within the scope of a single workstream.
+- A PR should only add/change content related to the issue associated with the PR.
 
 <a name="approvals"></a>
-### Approval Process
+### Pull Requests Approval Process
 
 - Once a PR is created, it needs to get the following approvals before it is merged into master.
-  - From at least one of committers for that workstream
+  - From at least one of contributor for that workstream
     - Approvers should be distributed equally among operators and vendors.
   - From WSL (to make sure consensus is reached).
   - WSL can request additional approvals reviews.
     - Recommendation is to use 4 approvals for complex PRs.
-  - 2 business days cool off period will be applied before Final approval.
+  - 2 business days cool off period should be applied before Final approval.
   - Final Approval by TSL (to make sure process is followed)
-- Only One approval is permitted per each organization for a given PR.
-- The selection of which Committers to approve a PR is made by the WSL and should take those factors into consideration:
-  - Committers needs to be actively discussing the PR to be selected for approval.
-  - Committers needs to be active in the workstream.
+- Only One approval will be counted per each organization for a given PR.
+- The selection of which contributor to approve a PR is made by the WSL and should take those factors into consideration:
+  - Contributor needs to be actively discussing the PR to be selected for approval.
+  - Contributor needs to be active in the workstream.
 - PRs will be merged automatically online by the TSL once consensus is reached and all approvals are received. 
-- If WSL is the person who is creating the PR, They need to request an alternate approver, preferably a co-lead or from the committers list.
-
+- If WSL is the person who is creating the PR, They need to request an alternate approver, preferably a co-lead or from the contributors list.
 
 <p align="right"><img src="artefacts//figures/approval_process.png" alt="scope" title="Scope" width="100%"/></p>
 <p align="center"><b>Figure 1:</b> Approval Process</p>
 
+<a name="idle"></a>
+### Idle Pull Requests Policy
+
+Pull Requests will be automatically labled as "**Idle**" when:
+- No Engagement from the author of the PR to address Review Requests or input from the community for more than 15 calender days.
```

Comment:
what kind of "input" from the community? Can we simplify this and the following rule into simply, "No engagement/activity (content, reviews, conversations) from author, reviewers and workstream contributors on non-merge ready PRs for 15 calendar days."

Reply:
The label should be set "automatically" ... what does Github allow here?

Reply:
clarified automation for @hdamker and fixed type based on @pgoyal01 feedback